dynawo/sources/Solvers/FixedTimeStep/DYNSolverCommonFixedTimeStep.cpp

```diff
@@ -7,20 +7,21 @@
 // file, you can obtain one at http://mozilla.org/MPL/2.0/.
```

Comment:
Copyright: this is a new file so it should be "// Copyright (c) 2021, RTE (http://www.rte-france.com)"